main.js

```diff
@@ -11,31 +11,41 @@ function generateNumbers() {
     return allNumbers;
 }
 
-function displayNumbers(allNumbers) {
-    let numbersOnScreen = allNumbers.length;
-
+function displayNumbers() {
+    const randomNumbers = generateNumbers();
+    const sortedNumbers = randomNumbers.slice(0).sort();
```

Comment:
This will not sort the array properly in some cases. If the random numbers array is lets say: [52, 18, 4, 90, 77]; This will result in [18, 4, 52, 77, 90]. 


-Max and I found this out as we initially wanted the generate numbers array to return a  sorted array but decided against it.

Reply:
Didn't know this, we'll use a comparative sort instead.